File: nodejs/loccount/ui.js
const sumLoc = {
    filename: "",
    total: 0,
    loc: 0
};

exports.ShowLoc = function (locstat) {
    console.log(locstat.filename + " " + locstat.total + " " + locstat.loc);
    sumLoc.total += locstat.total;
    sumLoc.loc += locstat.loc;
};

exports.ShowSum = function () {
    console.log();
    console.log("Sum:");
    console.log("  Total: " + sumLoc.total)
    console.log("  LOC  : " + sumLoc.loc)
};

File: csharp/loccount/loccount/loccount/Ui.cs
using System;
using System.Collections.Generic;
using System.Linq;

namespace loccount
{
    public class Ui
    {
        public static void Show(IEnumerable<LOCstat> locStats) {
            foreach (var locStat in locStats) {
                Console.WriteLine($"{locStat.Filename} {locStat.Total} {locStat.Loc}");
            }
            Console.WriteLine();            
            Console.WriteLine("Sum:");            
            Console.WriteLine($"  Total: {locStats.Sum(l => l.Total)}");            
            Console.WriteLine($"  LOC:   {locStats.Sum(l => l.Loc)}");            
        }
    }
}